github/tests/Organization.py

```diff
@@ -126,6 +129,15 @@ def testGetEvents(self):
     def testGetTeams(self):
         self.assertListKeyEqual(self.org.get_teams(), lambda t: t.name, ["Members", "Owners"])
 
+    def testCreateHookWithMinimalParameters(self):
+        hook = self.org.create_hook("web", {"url": "http://foobar.com"})
+        self.assertEqual(hook.id, 257967)
+
+    def testCreateHookWithAllParameters(self):
+        hook = self.org.create_hook("web", {"url": "http://foobar.com"}, ["fork"], False)
+        self.assertTrue(hook.active)  # WTF
```

Comment:
What is this? 😄

Reply:
I am not too sure 😂 The Respository.py test had something similar. I'll remove it, as it's not actually useful.

Reply:
I mean the comment